libecole/src/observation/nodebipartite.cpp

```diff
@@ -85,12 +85,14 @@ static auto extract_row_feat(scip::Model const& model) {
 
 	auto extract_row = [n_lps, obj_l2_norm](auto& iter, auto const row, bool const lhs) {
 		value_type const sign = lhs ? -1. : 1.;
-		value_type const row_l2_norm = static_cast<value_type>(row.l2_norm());
+		value_type row_l2_norm = static_cast<value_type>(row.l2_norm());
+		if (row_l2_norm == 0) row_l2_norm = 1.;
```

Comment:
Thanks. Should the change also be done for the columns features as well?
https://github.com/ds4dm/ecole/blob/170e7dab562164b234c96c8483c2f8d0fc3d14dd/libecole/src/observation/nodebipartite.cpp#L35

Reply:
You mean setting it equal to 1 if it's equal to zero? The `get_obj_norm` function you coded already does that, no?

https://github.com/ds4dm/ecole/blob/170e7dab562164b234c96c8483c2f8d0fc3d14dd/libecole/src/observation/nodebipartite.cpp#L26-L29

Reply:
Ha... Guess we need to make it similar to the other one then.